gui/slick/views/home_popularShows.mako

```diff
@@ -1,6 +1,10 @@
 <%inherit file="/layouts/main.mako"/>
 <%!
     from sickbeard.helpers import anon_url
+    from sickbeard import db    
+    myDB = db.DBConnection()
+    sql_statement  = 'SELECT imdb_id from tv_shows where imdb_id is not null'
+    sql_result = myDB.select(sql_statement)    
```

Comment:
I didn't test this, but it may prevent an extra SQL query:

```python
import sickbeard

imdb_ids = [show['imdbid'] for show in sickbeard.showList if show['imdbid']]
```

And line 34 would become: `% if cur_result['imdb_tt'] not in imdb_ids:`